doc/examples/template.py

```diff
@@ -0,0 +1,168 @@
+# template.py - template file for python-control module
+# RMM, 3 Jan 2024
+
+"""Template file for python-control module.
+
+This file provides a template that can be used when creating a new
+file/module in python-control.  The key elements of a module are included
+in this template, following the suggestions in the Developer Guidelines.
+
+The first line of a module file should be the name of the file and a short
+description.  The next few lines can contain information about who created
+the file (your name/initials and date).  For this file I used the short
+version (initials, date), but a longer version would be to do something of
+the form::
+
+  # filename.py - short one line description
```

Comment:
I did not search for a PEP or public style guide about this, but to me, this first line comment is too redundant:

* The first line of the module-level docstring already is supposed to have a short one line description.
* Including the name of the file inside the file is not informative (presumably the user knows which file they are reading...) and makes refactoring later more difficult (change file name also requires changing this comment).

That said, many (but not all) source files in python-control follow this first line convention, so OK to keep it in this template if you want the template to reflect facts on the ground or if this is still your preferred style.

Reply:
Agree it is redundant, but this has been my preferred style for the first line of a file for 30 years (old habits die hard -:).  Leaving this one in place for now.